### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by rapi**
Add storybook, eslint, vue-cli-3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "vue-crafted-table-bak",
```
Comment (andreikojusko) [inline package.json]: Why `-bak` ?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
```
Comment (pcraciun) [inline public/index.html]: is it a JSP code? not sure we need it here

File: package.json (hunk 1 above)
Comment (rapi) [reply package.json]: You are right

File: public/index.html (hunk 2 above)
Comment (rapi) [reply public/index.html]: yes, you are right

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>vue-crafted-table-bak</title>
```
Comment (pcraciun) [inline public/index.html:7]: -bac also here

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+  </head>
+  <body>
+    <noscript>
+      <strong>We're sorry but vue-crafted-table-bak doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```
Comment (pcraciun) [inline public/index.html:12]: -bac also here

**Review by pcraciun — APPROVED** [review]

**Review by andreikojusko — APPROVED** [review]

**Pull request merged**